fin-wallet-api/src/main/java/com/codegym/finwallet/service/impl/ImageServiceImpl.java

```diff
@@ -0,0 +1,106 @@
+package com.codegym.finwallet.service.impl;
+
+import ch.qos.logback.classic.encoder.JsonEncoder;
+import com.codegym.finwallet.constant.VarConstant;
+import com.codegym.finwallet.dto.CommonResponse;
+import com.codegym.finwallet.dto.payload.response.ProfileResponse;
+import com.codegym.finwallet.entity.Profile;
+import com.codegym.finwallet.repository.ProfileRepository;
+import com.codegym.finwallet.service.ImageService;
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class ImageServiceImpl implements ImageService {
+    private final ModelMapper modelMapper;
+    private final ProfileRepository profileRepository;
+    private final String UPDATE_PROFILE_AVATAR_SUCCESS_MESSAGE = VarConstant.UPDATE_PROFILE_AVATAR_SUCCESS;
```

Comment:
should rename to VarConstant.USER_PROFILE_AVATAR_SUCCESS and use it at the line of code 93